resources/quality/jgaurora_a6/jgaurora_a6_0.16_optimal.inst.cfg

```diff
@@ -20,7 +20,6 @@ speed_travel = 80
 retraction_amount = 4
 adhesion_type = skirt
 acceleration_enabled = True
-adhesion_type = skirt
```

Comment:
Don't ask me how I found this change amidst 3000 files... Looked wrong to me but showing the context here in Github I see that it is duplicate and removing this line was correct.